test/local/metrics/context.js

```diff
@@ -697,6 +697,37 @@ describe('metricsConext', () => {
       var mockRequest = {
         headers: {
           'user-agent': 'test-agent'
+        }
+      }
+
+      var metricsContext = require('../../../lib/metrics/context')(mockLog, mockConfig)
```

Comment:
Think the test case name above should be called `metricsContext.validate with missing payload`.